app/frontend/styles/components/jobs-banner.scss

```diff
@@ -0,0 +1,11 @@
+.jobs-banner {
+  background-color: govuk-colour('light-grey');
+  left: 50%;
+  margin-bottom: govuk-spacing(3);
+  margin-left: -50vw;
+  margin-right: -50vw;
```

Comment:
usually i like to put it all in one line if there are more than 2 properties `margin: 0 -50vw govuk-spacing(3) -50vw` but no biggie.

Reply:
I'll try it out on my other branch (I just forget every time where it starts and which direction it goes 😂 )